users/pcoves/pcoves.c

```diff
@@ -0,0 +1,42 @@
+#include "pcoves.h"
+
+#ifdef RAINBOW_UNICORN_ENABLE
+#include "rainbowUnicorn.h"
+#endif
+
+#ifdef UNICODE_ENABLE
+#include "unicode.h"
+#endif
+
+#if SECRET_ENABLE
+#include "secret.h"
+#endif
+
+void eeconfig_init_keymap(void) { return; }
+
+void eeconfig_init_user(void) {
+    set_unicode_input_mode(UC_LNX);
+    eeconfig_init_keymap();
+}
+
+bool process_record_keymap(uint16_t keycode, keyrecord_t *record) { return true; }
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case AUTRUCHE:
+            if (record->event.pressed) SEND_STRING("Autruche");
+            return false;
```

Comment:
You should return true unless you're replacing functionality. 
```suggestion
            return true;
```